cmd/client_changefeed.go

```diff
@@ -243,6 +243,13 @@ func verifyChangefeedParamers(ctx context.Context, cmd *cobra.Command, isCreate
 		return nil, errors.Annotate(err, "can not load timezone, Please specify the time zone through environment variable `TZ` or command line parameters `--tz`")
 	}
 
+	if sortEngine == string(model.SortInFile) {
+		err = util.IsDirAndWritable(sortDir)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
I think it should be checked on CDC server.

Reply:
addressed, check on cdc server is more suitable.